UndertaleModLib/UndertaleIO.cs

```diff
@@ -344,7 +344,9 @@ private bool ProcessCountExc(uint poolSize = 0)
             if (poolSize != 0 && poolSize != objectPool.Count)
             {
                 SubmitWarning("Warning - the estimated object pool size differs from the actual size.\n" +
-                              "Please report this on UndertaleModTool GitHub.");
+                              "Please report this on UndertaleModTool GitHub.\n" +
+                              "Unless you have a .win with SWF sprites,\n" +
+                              "since YYSWF objects are not pooled due to overlapping.");
```

Comment:
Are you sure that there could be the pool size mismatch? All the SWF related classes are not in the pool, so is that really possible?